### DISCUSSION TITLE
NDA package/eprime

### DISCUSSION THREAD

**Discussion by sanaashrafali** [now-i-know-my-abcd/docs#13] [Where in the World is…? (Help)] [answered]
What NDA package has the eprime files for release 3?

**Comment by sanaashrafali** [comment, marked as answer]
Mriqcrp302

**Reply by fhaist** [reply]
The DAIRC emphatically suggests that current releases be used (i.e., 4.0). The latest releases contain all prior data and current data fixes. It is the same package name in 4.0.
